hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -34,7 +35,7 @@ class ActionEvaluatorConfig:
     reactions_queue_url: str
 
     @classmethod
-    @lru_cache(maxsize=1)
+    @lru_cache(maxsize=None)
```

Comment:
I saw a change from `1` to `None` in another PR. Asked Barrett about it and confirmed `None`'s the way to go.